app/helpers/value_helper.rb

```diff
@@ -69,9 +69,14 @@ def links_to_collections(args)
     end.join('<br>').html_safe
   end
 
+  # Discussed removing this method because the field should not be modified.
   def value_for_originInfo_date_created_tesim(args)
```

Comment:
Since this method isn't doing anything specifically related to `orginInfo_date_created_tesim` it would be nice to see it renamed and added explicitly using `helper_method` in `catalog_controller.rb` https://github.com/sul-dlss/argo/blob/develop/app/controllers/catalog_controller.rb#L49 https://github.com/projectblacklight/blacklight/wiki/Configuration---Results-View#helper-method